### ISSUE TITLE
Question About Training Procedure WRT Input and Labels

### ISSUE THREAD

**Issue by eschaffn** [gjmulder/baize-chatbot#2]
Looking through the code in your fork, and the original, it seems like when using data with the format `{"input": "some text", "output": "some output"}` the training code only uses the "input" to generate the prompt and then passes these to the model. Is this fine-tuning using CLM on just the inputs? I don't see the outputs being tokenized or fed to the model as labels anywhere to calculate loss and backpropagate. 

I may have been confused in thinking that this was for supervised fine-tuning, is this not the case? 

Thank you!

**Comment by eschaffn** [comment]
It seems like just concatenating my inputs and ouputs together may be similar to what you're doing, does this sound right?
